src/material-experimental/mdc-dialog/dialog.scss

```diff
@@ -0,0 +1,60 @@
+@import '@material/dialog/mixins.import';
+@import '@material/dialog/variables.import';
+@import '../mdc-helpers/mdc-helpers';
+@import './mdc-dialog-structure-overrides';
+
+// Dialog content max height. This has been copied from the standard dialog
+// and is needed to make the dialog content scrollable.
+$mat-dialog-content-max-height: 65vh !default;
+// Dialog button horizontal margin. This has been extracted from MDC as they
+// don't expose this value as variable.
+$mat-dialog-button-horizontal-margin: 8px !default;
+
+@include mdc-dialog-core-styles($query: $mat-base-styles-query);
+@include _mdc-dialog-structure-overrides();
+
+// The dialog container is focusable. We remove the default outline shown in browsers.
+.mat-mdc-dialog-container {
+  outline: 0;
+}
+
+// MDC sets the display behavior for title and actions, but not for content. Since we support
+// using the `mdc-dialog__content` as custom element, we need to set the element to `block`.
+.mat-mdc-dialog-content {
+  display: block;
+}
+
+.mat-mdc-dialog-actions {
+  // For backwards compatibility, actions align at start by default. MDC usually
+  // aligns actions at the end of the container.
+  justify-content: start;
```

Comment:
Should we change the default to match MDC and allow `[align='start']` to override? It's just a visual change so it seems like it should be ok

Reply:
That's something I'm not too sure about. Maybe your or @jelbourn have some strong thoughts about this? I certainly could imagine `end` being the default now.

Potentially we could incorporate the old behavior into the legacy padding mixin (and rename it), or we create another one for legacy action alignment?

Reply:
I actually had a PR to do this a long time ago, but we never merged it because of the potential for screenshot diff failures https://github.com/angular/components/pull/13620.

Reply:
Good to know. We are already having a breaking change in the MDC-based dialog (i.e. no outer padding), so we might be able to have that as a breaking change too?

Alternatively, as proposed above, we could add the previous default alignment to a legacy opt-in mixin.

Reply:
I agree that leaving the default the same as it used to be will make this easier to roll out; we can change that in the futre if we really want to